docs-2.0/3.ngql-guide/8.clauses-and-options/return.md

```diff
@@ -168,12 +158,11 @@ nebula> MATCH (v:player{name:"Tim Duncan"})-[e]->(v2) \
 +-----------------+----------+----------+
 | "Spurs"         | "serve"  | __NULL__ |
 +-----------------+----------+----------+
-Got 3 rows (time spent 2976/3658 us)
 ```
 
 ## Return expression results
 
-To return the results of expressions such as literals, functions, or predicates, set them in a `RETURN` clause.
+To return the results of expressions such as characters, functions, or predicates, set them in a `RETURN` clause.
```

Comment:
Cancel the update to this line. The source doc is inaccurate, I fixed it in https://github.com/vesoft-inc/nebula-docs-cn/pull/807.